libs/fixed_point/README.md

```diff
@@ -7,7 +7,7 @@
 
 # Overview
 
-The Fixed Point Number library provides a library to use fixed-point numbers in Sway. It has 3 distinct unsigned types: `UFP32`, `UFP64` and `UFP128` as well as a signed type - `IFP64`. This type is stack allocated.
+The Fixed Point Number library provides a library to use fixed-point numbers in Sway. It has 3 distinct unsigned types: `UFP32`, `UFP64` and `UFP128` as well as 2 signed types `IFP64` and `IFP256`. These types are stack allocated.

```

Comment:
https://github.com/FuelLabs/sway-libs/pull/105 will cause merge conflicts with this. Will wait to review this until it's merged first

Reply:
This needs to be updated before we can merge